syntax/std_seq.go

```diff
@@ -64,5 +65,108 @@ func stdSeq() rel.Attr {
 	return rel.NewTupleAttr("seq",
 		rel.NewNativeFunctionAttr("concat", stdSeqConcat),
 		rel.NewNativeFunctionAttr("repeat", stdSeqRepeat),
+		createNestedFuncAttr("contains", 2, func(args ...rel.Value) rel.Value {
+			switch args[1].(type) {
+			case rel.String:
+				return rel.NewBool(strings.Contains(mustAsString(args[1]), mustAsString(args[0])))
+			case rel.Array:
+				return arrayContains(args[1].(rel.Array), args[0])
+			case rel.Bytes:
+				return rel.NewBool(strings.Contains(args[1].String(), args[0].String()))
+			}
+
+			return rel.NewBool(false)
+		}),
+		createNestedFuncAttr("has_prefix", 2, func(args ...rel.Value) rel.Value {
+			switch args[1].(type) {
+			case rel.String:
+				return rel.NewBool(strings.HasPrefix(mustAsString(args[1]), mustAsString(args[0])))
+			case rel.Array:
+				return arrayPrefix(args[1].(rel.Array), args[0])
+			case rel.Bytes:
+				return rel.NewBool(strings.HasPrefix(args[1].String(), args[0].String()))
+			}
+
+			return rel.NewBool(false)
```

Comment:
Yes, it is from code rules. And the concern is it will stop the process if it panic, and it is not very good. In this case, return false can make process handler more cases like unsupported data structure and will not violate `has_suffix` rule still.